tools/scripts/mcufla.sh

```diff
@@ -109,10 +123,47 @@ build_prerequisites() {
 	fi
 }
 
-parse_input $@
+build_prerequisites_stm32() {
+	local HALF_CORES=$(expr $(grep -c '^processor' /proc/cpuinfo) / 2)
+
+	if [ ! -d ~/.stm32/openocd ]; then
+		git clone https://github.com/STMicroelectronics/OpenOCD.git ~/.stm32/openocd &&
+		pushd ~/.stm32/openocd &&
+		./bootstrap &&
+		./configure &&
+		make SHELL='sh' -j$HALF_CORES &&
+		popd
 
-build_prerequisites
+		if [ $? -ne 0 ]; then
+			rm -rf ~/.stm32/openocd
+			exit
+		fi
+	fi
+}
 
-print_run ~/.maxflash/openocd/src/openocd -s ~/.maxflash/openocd/tcl \
-	  -c "adapter driver cmsis-dap; transport select swd; $DAP_SERIAL" \
-	  -f "target/$TARGET.cfg"  -c "program $1 verify reset exit"
+configure_specific_platform() {
+	if [ $PLATFORM == 'maxim' ]; then
+		echo "Maxim platform detected"
+		build_prerequisites_maxim
+		print_run ~/.maxflash/openocd/src/openocd -s ~/.maxflash/openocd/tcl \
```

Comment:
`maxflash` -> `mcuflash` throughout the file

Reply:
bad phrasing, sorry, I meant to say we should use `~/.mcuflash/` folder to store openocd sources and builds